README.md

```diff
@@ -61,7 +61,9 @@ When an issuer.example context wants to provide tokens to a user (i.e. when the
 ```
 fetch('<issuer>/.well-known/trust-token', {
   trustToken: {
-    type: 'token-request',
+    type: '0xFF53',
+    version: '1',
```

Comment:
Another option is making "type" a complex type that includes both the type and version as part of it (type: {trustToken: 1} or type: ["trust-token", 1], etc).